contracts/staking/test/actors/staker_actor.ts

```diff
@@ -31,19 +29,20 @@ export class StakerActor extends BaseActor {
         // check balances
         const expectedStakerBalances = initStakerBalances;
         expectedStakerBalances.zrxBalance = initStakerBalances.zrxBalance.minus(amount);
-        expectedStakerBalances.stakeBalance = initStakerBalances.stakeBalance.plus(amount);
         expectedStakerBalances.stakeBalanceInVault = initStakerBalances.stakeBalanceInVault.plus(amount);
-        expectedStakerBalances.activatedStakeBalance = initStakerBalances.activatedStakeBalance.plus(amount);
+        expectedStakerBalances.activeStakeBalance.current = initStakerBalances.activeStakeBalance.current.plus(amount);
+        expectedStakerBalances.activeStakeBalance.next = initStakerBalances.activeStakeBalance.next.plus(amount);
         await this.assertBalancesAsync(expectedStakerBalances);
         // check zrx balance of vault
         const finalZrxBalanceOfVault = await this._stakingWrapper.getZrxTokenBalanceOfZrxVaultAsync();
         expect(finalZrxBalanceOfVault).to.be.bignumber.equal(initZrxBalanceOfVault.plus(amount));
```

Comment:
Nit: Add descriptions to expects across the whole file for consistency and easier debugging. It may even be good to pass the calling functions name into `assertBalancesAsync` so that the error messages are even nicer when `assertBalancesAsync` is called by other functions in the actor.